src/main/kotlin/mate/academy/figure/RightTriangle.kt

```diff
@@ -0,0 +1,18 @@
+package mate.academy.figure
+
+import mate.academy.model.Color
+
+class RightTriangle(
+    color: Color,
+    private val firstLeg: Double,
+    private val secondLeg: Double) : Figure(color) {
+    override fun draw() {
+        println("Figure: right triangle, "
+                + "area: ${getArea()} sq. units, "
```

Comment:
You may use triple quotes marks  with .trimIndent() extension to avoid concatenation